.github/workflows/CI.yml

```diff
@@ -485,6 +485,9 @@ jobs:
           aws-secret-access-key: ${{ secrets.INTEG_TEST_AWS_KEY_SECRET }}
           aws-region: us-west-2
 
+      - name: Set up QEMU
```

Comment:
Should probably only run on cache miss. Also, nitpick, but maybe keep the buildx and qemu set up steps together. Maybe move them both to right before the build-push action.